main/tests/server/src/com/google/refine/operations/recon/ReconOperationTests.java

```diff
@@ -82,7 +83,7 @@ public class ReconOperationTests extends RefineTest {
             "           \"columnName\" : \"researcher\",\n" +
             "           \"expression\" : \"forNonBlank(cell.recon.judgment, v, v, if(isNonBlank(value), \\\"(unreconciled)\\\", \\\"(blank)\\\"))\",\n"
             +
-            "           \"name\" : \"researcher: judgment\"\n" +
+            "           \"name\" : \"researcher: " + OpenRefineMessage.recon_operation_judgement_facet_name() + "\"\n" +
```

Comment:
This needs escaping to make sure the JSON syntax is still valid.

Reply:
Ok, I escaped the strings with `StringEscapeUtils.escapeJson()`.